plugins/services/src/js/service-configuration/PodNetworkConfigSection.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import ConfigurationMapHeading from '../../../../../src/js/components/ConfigurationMapHeading';
+import ConfigurationMapKeyValueRow from '../components/ConfigurationMapKeyValueRow';
+import ConfigurationMapSection from '../../../../../src/js/components/ConfigurationMapSection';
+import ConfigurationMapTable from '../components/ConfigurationMapTable';
+import ConfigurationVisible from '../components/ConfigurationVisible';
+import ServiceConfigDisplayUtil from '../utils/ServiceConfigDisplayUtil';
+
+module.exports = ({appConfig}) => {
+  let endpoints = (appConfig.containers || []).reduce((memo, container) => {
+    return memo.concat(
+      (container.endpoints || []).map((endpoint) => {
+        let lbAddress = null;
+        lbAddress = Object.keys(endpoint.labels || {}).reduce((memo, label) => {
+          if (label.startsWith('VIP_')) {
+            let [prefix, port] = endpoint.labels[label].split(':');
+            memo.push(`${prefix}.marathon.lb4lb.thisdcos.directory:${port}`);
+          }
+          return memo;
+        }, []);
+
+        return {
+          name: endpoint.name,
+          protocol: endpoint.protocol,
+          port: endpoint.containerPort,
+          lbAddress: lbAddress.join(', '),
+          container: ServiceConfigDisplayUtil.getContainerNameWithIcon(container)
+        };
+      })
+    );
+    return memo;
+  }, []);
+
+  return (
+    <ConfigurationVisible when={endpoints}>
+      <ConfigurationMapHeading level={1}>Network</ConfigurationMapHeading>
+      <ConfigurationMapSection key="pod-general-section">
+
+        {/* General section */}
+        <ConfigurationMapKeyValueRow
+          label="Network Type"
+          placeholder={<em>Unknown</em>}
+          map={{
+            'container': 'Container',
+            'host': 'Host'
+          }}
+          value={appConfig.networks && appConfig.networks[0] &&
```

Comment:
you can use: `value={findNestedPropertyInObject(appConfig, 'networks.0.mode')}` for this